app/controllers/submissions_controller.rb

```diff
@@ -1,11 +1,16 @@
 class SubmissionsController < ApplicationController
   def new
     @submission = Submission.new
+    @submission.submission_packets.build.build_poem
   end
 
   def create
-    @submission = Submission.create(submission_params_with_user)
+    @submission = current_user.submissions.build(submission_params)
+    @submission.submission_packets.map do |packet|
```

Comment:
:+1: noted. Confused "making a change to the object" for "capturing the result".